src/components/molecules/payment-detail/index.tsx

```diff
@@ -0,0 +1,26 @@
+//HELPERS
+import { numberWithCommas } from "src/core/helpers";
```

Comment:
In a compiled solution using typescript this is still bad practice and I would recommend replacing this with aliases or relative imports with the base entry point to `src` directory.

These are just suggestions so it can be ignored 🎩